apps/style/common.scss

```diff
@@ -266,22 +267,30 @@ button.arrow:disabled {
     padding-inline-end: 9px;
   }
 }
+#soft-buttons.hidden {
+  display: none;
+}
 
 .soft-buttons-1 {
-  display: table-cell;
-  vertical-align: top;
+  @supports not (gap: 9px) {
+    min-width: 43px;
```

Comment:
where do these `min-width` numbers come from? could we add something like a `margin-left` to each child instead?

Reply:
The numbers are basically 43n+9(n-1) where n is the number of buttons. 43 is the button size and 9 is the gap between. This isn't introducing new tech debt so much as reverting back to previous tech debt while we wait for our browser support to catch up. We had similar hard-coded numbers for these exact rules for years until I took them out when I started working in this space (https://github.com/code-dot-org/code-dot-org/pull/44080/files).

I don't think we can just add a margin to one side of the buttons. If we do this to all buttons, it will result in an extra margin on one side or the other (the set of buttons will wrap with other elements in the larger flexbox), and we've already seen how difficult it is to try to make an exception for one button. Whether we went with `margin-left`, `margin-inline-start`, or anything similar we'd still probably end up needing to go back down the react/redux roads in order to address this. My preference would be that we keep it as is, because it's quite similar to how it has always worked but it also gives a strong signal to how we want to move in the future. I could potentially add a comment to explain the numbers. 

Thoughts?

Reply:
ahhh, i forgot we previously had these widths hardcoded -- i agree that this approach is fine in that case. we could either add comments, or use variables to calculate these widths to make it clearer what the numbers mean. since we plan to delete this code, i think comments work, but if it were permanent code, i'd prefer variables/calculations